cuebot/src/main/resources/public/dtd/cjsl-1.9.dtd

```diff
@@ -29,15 +29,15 @@
     cores                   NMTOKEN                     #REQUIRED
```

Comment:
Similarly, this should become a new `1.10` version instead of modifying the existing version.